<launch>
  <arg name="foreign_ip" default="192.168.212.6"/>  
  <arg name="foreign_port" default="11311" />
  <arg name="node_name" default="node_robot1" /> <!-- must be different with robot2 -->

  <param name="foreign_ip"  value="$(arg foreign_ip)"/>
  <param name="foreign_port"  value="$(arg foreign_port)"/>
  <param name="connectCheck_Hz"  value="200"/>
  <param name="msgsFrequency_Hz"  value="10" />
  <param name="publish_name" value="rdvremote" />
  <node pkg="multimaster_example" type="multimaster_odom" name="$(arg node_name)" output="screen">
  </node>

</launch>
